shell/packages/sandstorm-accounts-ui/login_buttons.js

```diff
@@ -2,6 +2,9 @@
 var loginButtonsSession = Accounts._loginButtonsSession;
 
 var helpers = {
+  isCurrentRoute: function (routeName) {
+    return Router.current().route.getName() == routeName;
```

Comment:
Well, we're aiming to move off of iron router, so ideally we design some abstract interface that provides the functionality we want and pass that down -- something that we can re-implement in terms of some other router later.